src/nvim/ex_docmd.c

```diff
@@ -76,6 +76,16 @@
 #include "nvim/event/rstream.h"
 #include "nvim/event/wstream.h"
 
+/// The scope of a command.
+///
+/// The higher a number, the deeper a scope.
+enum {
+  SCOPE_EDITOR,     ///< Affects the entire instance of NeoVim.
+  SCOPE_TABPAGE,    ///< Affects on tab page.
+  SCOPE_WINDOW,     ///< Affects one window.
+  NUMBER_OF_SCOPES  ///< Total number of scoped defined.
```

Comment:
Why not just `#define` `NUMBER_OF_SCOPES`?  It should probably be named `MAXCDSCOPE`, too.